src/components/Timer/Timer.scss

```
@@ -1,61 +1,141 @@
 @import "constants/style.scss";
 
+$timer-display-height: 42px;
+$timer-display-width: 114px;
+$timer__content-height: 38px;
+$timer__short-actions-height: 28px;
+
 .timer {
+  height: $timer-display-height;
+  width: $timer-display-width;
+  color: $color-white;
+  background-color: $menu-icon-background-color--dark;
+  border-radius: calc($timer-display-height / 2);
   position: relative;
-  width: auto;
-  display: grid;
-  grid-template-columns: 1fr auto 1fr;
+  border: 2px solid $menu-icon-background-color--dark;
 }
-
-.timer > * {
-  color: $color-white;
-  background-color: $color-backlog-blue;
-  vertical-align: bottom;
-  box-shadow: 0 8px 16px #0057ff25;
+.timer > svg {
+  position: absolute;
+  height: 100%;
+  background-color: $menu-icon-background-color--dark;
+  border-radius: 100%;
 }
-
-[theme="dark"] {
-  .timer > * {
-    background-color: $color-planning-pink;
-    box-shadow: 0 8px 16px #e2036025;
-  }
+[theme="dark"] .timer > svg {
+  background-color: $color-white;
+  color: $color-black;
 }
 
 .timer > span {
-  grid-column: 2;
-  display: inline-block;
-  border-radius: 18px;
-  width: 84px;
-  cursor: default;
-  font-weight: bold;
-  line-height: $line-height--large;
+  position: absolute;
+  right: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  padding-left: $timer__content-height;
+  line-height: $timer__content-height;
   text-align: center;
+  font-weight: bold;
+}
+
+.timer__progress-bar {
+  position: absolute;
+  top: 0;
+  left: 19px;
```

```suggestion
  left: calc($timer__content-height / 2);
```

maybe `calc($timer__content-height / 2)` could be a new variable as well, you use it quite often